### PR TITLE
Prevent sidebar from growing and shrinking in IE

### LINKED ISSUE #1434: Inconsistent Stickiness Sidebar Behavior with Internet Explorer
#1430 appears to work fine in Chrome, but has some interesting behavior on Internet Explorer.

1) Sidebar resizes on scroll

APD Load: 

APD on scroll:

2) ~~Page content is not centered in IE~~ Not sure when this was fixed, but IE content is centered.

### This task is done when...
 - [ ] Left side navigation remains consistent in size during scrolling in IE
 - [ ] The sidebar has appropriate bottom whitespace when the page is scrolled to the bottom (if possible - see [the first image](https://github.com/18F/cms-hitech-apd/pull/1721#issuecomment-524382855) to see the problem
 - [ ] ~~Page content is centered in IE~~

### PR DESCRIPTION
**Pull request by mgwalker**
### This pull request changes...

- sets the sidebar to a fixed width on IE, so it doesn't get wider and narrower and weird stuff
- closes #1434

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] Changelog is updated as appropriate

### This feature is done when...

- [ ] Product has approved the experience

### CODE REVIEW

**Comment by mgwalker** [comment]
This creates an `isIE()` utility function, and then calls that in the sidebar code to set a conditional class when we're running in IE. @hbillings I'm wondering if you think it'd make sense to set a conditional class on the `<body>` when we're running in IE instead, and we could separate out IE-specific hacks in our Sass?

**Comment by mgwalker** [comment]
slapping forehead

Yep, this happens when the browser is narrow enough that the sidebar's native width is _less than_ the fixed width. This is so frustrating! I'm not sure what to do about that - my knee-jerk reaction is to add some Javascript to compute the proper width, but that's so gross. I'm going to wait until next week and ask Heather because she's smart.

**Comment by hbillings** [comment]
MEDIA QUERIES!

But, also, I'm wondering if instead of setting the fixed width on the sidebar itself, if we could set it on the interior items. Or maybe on both. What's happening is that when `position: fixed` is applied, the width calculation changes (which is why @jeromeleecms has been seeing the width change -- `position: fixed` isn't applied in the initial scroll position, but it is once the page is scrolled down). If we set the width on a child of the container that gets `position: fixed` applied to it, I think we'd maybe see more predictable behavior. I'll try to get my IE setup working tomorrow and try a few things out.

File: web/src/containers/Sidebar.js [hunk 1]
```diff
@@ -3,18 +3,14 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
 import VerticalNav from '@cmsgov/design-system-core/dist/components/VerticalNav/VerticalNav';
-import stickybits from 'stickybits';
 
 import { t } from '../i18n';
 import { jumpTo } from '../actions/navigation';
+import { isIE } from '../util';
```
Comment (hbillings) [inline web/src/containers/Sidebar.js]: We don't need this here any more, right?
```suggestion
```

File: web/src/containers/Sidebar.js [hunk 2]
```diff
@@ -200,28 +196,24 @@ class Sidebar extends Component {
...
-                  height="40"
-                />
-              )}
-              <h1>{place.name}</h1>
-            </div>
-            <VerticalNav
-              selectedId={activeSection || 'apd-state-profile-overview'}
-              items={links}
-            />
+      <aside className={`site-sidebar${isIE() ? ' site-sidebar__ie' : ''}`}>
```
Comment (hbillings) [inline web/src/containers/Sidebar.js]: I think we can just do...
```suggestion
      <aside className="site-sidebar">
```

File: web/src/util/index.js [hunk 3]
```diff
@@ -267,3 +267,14 @@ export const replaceNulls = (obj, newValue = '') => {
   replace(objNew);
   return objNew;
 };
+
+/**
+ * Discover whether the current browser is a version of Internet Explorer
+ * @returns {Boolean} True if the browser is IE, false otherwise. Returns false
+ *                    for Edge.
```
Comment (hbillings) [inline web/src/util/index.js]: Can we get rid of this also, or does the save button need this?

**Pull request merged**